ceremonies/src/tests.rs

```diff
@@ -1123,6 +1124,17 @@ fn get_meetup_time_works(lat_micro: i64, lon_micro: i64) {
 			tol > (EncointerCeremonies::get_meetup_time(location).unwrap() as i64 - mtime as i64)
 				.abs() as u64
 		);
+
+		EncointerCeremonies::set_meetup_time_offset(
+			Origin::signed(master()),
+			Moment::from(100_000u64),
+		)
+		.ok();
+		assert!(
+			tol > (EncointerCeremonies::get_meetup_time(location).unwrap() as i64 -
+				(mtime + 100_000u64) as i64)
+				.abs() as u64
+		);
 	});
```

Comment:
Here, I wonder if we should do a separate test for that feature, but this would be quite some duplicate code currently for the test-setup... may be just leave it at that.